jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -1082,6 +1102,23 @@ class DeploymentFlow extends React.Component {
     return null;
   }
 
+  /**
+    Scroll the deployment flow to an element with an id that matches the
+    hash state.
+  */
+  _scrollDeploymentFlow() {
```

Comment:
I feel like the hash value should be passed to this as it could then be used as a more generic 'scroll to selector' method.